service/src/test/java/com/myhome/controllers/HouseControllerTest.java

```diff
@@ -147,12 +147,13 @@ void getHouseDetailsNotExists() {
   void listAllMembersOfHouse() {
     // given
     Set<HouseMember> testHouseMembers = TestUtils.HouseMemberHelpers.getTestHouseMembers(TEST_HOUSE_MEMBERS_COUNT);
-    Set<ListHouseMembersResponse.HouseMember> testHouseMemberDetails = testHouseMembers.stream()
-        .map(member -> new ListHouseMembersResponse.HouseMember(member.getMemberId(),
-            member.getName()))
+    Set<com.myhome.model.HouseMember> testHouseMemberDetails = testHouseMembers.stream()
+        .map(member -> new com.myhome.model.HouseMember().memberId(member.getMemberId()).name(member.getName())
```

Comment:
Can we reformat it in a more readable way?

Reply:
Done!